src/components/layout/Header/Nav.js

```diff
@@ -1 +1,76 @@
-export default () => null
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import Logo from '../../../../static/svg/yalla-logo.svg'
+import LogoBlack from '../../../../static/svg/yall-logo-black.svg'
+
+const MenuContainer = styled.div`
+  cursor: pointer;
+`
+
+const Nav = styled.nav`
+  display: flex;
+  justify-content: space-between;
+  padding: 50px;
+`
+const NavContainer = styled.div`
+  position: 'absolute';
+  top: 0;
+`
+
+const Span = styled.span`
+  position: relative;
+  margin-top: 3.5rem;
+  &,
+  &:after,
+  &:before {
+    mix-blend-mode: difference;
+    width: 3rem;
+    height: 2px;
+    background-color: ${props =>
+      props.menuColor === 'white' ? '#FFF' : '#000'};
+    display: inline-block;
+  }
+
+  &:after,
+  &:before {
+    content: '';
+    position: absolute;
+    left: 0;
+    transition: all 0.3s;
+  }
+
+  ,
+  &:after {
+    top: 0.8rem;
+  }
+  ,
+  &:before {
+    top: -0.8rem;
+  }
+`
+const LogoContainer = styled(MenuContainer)``
```

Comment:
u can just use it as is

Reply:
I meant to do this in case @jema28  complete it and want to complete this element after opening